pages/accountLists/[accountListId]/tools/appeals/index.page.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactElement } from 'react';
```

Comment:
The Given contacts should show the individual gift they gave, not their regular giving.
![Uploading Screenshot 2024-08-21 at 12.47.01 PM.png…]()

Reply:
Fixing this in a different PR